iac/terraform/aws/local.tf

```diff
@@ -7,4 +7,5 @@ locals {
     for lambda in local.all_lambda_metadata :
     lambda.invocation_arn_placeholder => lambda.lambda_invoke_arn
   })
+  apigateway-domain = var.environment == "prod" ? "api.${var.bloodconnect_domain}" : "${var.environment}-api.${var.bloodconnect_domain}"
```

Comment:
don't use strings like "prod". Instead move them to a central location and ref them with local.ENVIRONMENT.PROD so that it's clear that this is static

Reply:
var.environment == local.ENVIRONMENT.PROD

Reply:
please look into the following commit and let me know if the change is appropriate
https://github.com/CraftsmenLtd/BloodConnect/pull/119/commits/7c3524195c0f4c12914edb6aefeb6d3320b4a49f